### ISSUE TITLE
TPB and BPG

### ISSUE THREAD

**Issue by YifanC** [DUNE/larnd-sim#168]
I assume `TPB` is "track per batch" and `BPG` is "batch per gpu"?
Is the way we use `TPB` and `BPG` optimal at all in simulat_pixel.py? (redefining TPB and BPG in the middle of the simulation with hardcoded values and shape)

**Comment by krwood** [comment]
@mjkramer I wonder if Ronan might want to look into this?

**Comment by mjkramer** [comment]
Yes if @ronandoherty1 wants to look into optimizing TPB and BPG then that would be pretty awesome of him.
